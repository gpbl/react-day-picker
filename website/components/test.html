<!DOCTYPE html>
<html>
<head>
<style>
body {
  display: flex;
  margin: 0;
  padding: 0;
  min-height: 100vh;
  overflow-x: hidden;
}

.sidebar {
  position: sticky;
  top: 0;
  height: 100vh;
  width: 200px;
  background-color: #f5f5f5;
  padding: 20px;
  overflow-y: auto;
}

.main-content {
  flex-grow: 1;
  padding: 20px;
}
</style>
</head>
<body>

<div class="sidebar">
  <h2>Sidebar</h2>
  <p>This is a sticky sidebar.</p>
</div>

<div class="main-content">
  <h2>Main Content</h2>
  <p>Scroll the page.</p>
  <!-- Add enough content here so that the page scrolls -->
</div>

</body>
</html>
