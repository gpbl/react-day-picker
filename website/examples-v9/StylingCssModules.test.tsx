import { app, mockDate, renderApp } from "@/test";

import { CssModules } from "./CssModules";

const today = new Date(2021, 10, 25);
mockDate(today);

beforeEach(() => {
  renderApp(<CssModules />);
});

test("should match the snapshot", () => {
  expect(app()).toMatchSnapshot();
});
