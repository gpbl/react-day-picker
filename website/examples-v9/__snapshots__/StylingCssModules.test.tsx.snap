// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<div
  role="application"
>
  <div
    class="rdp-calendar"
    id=":r0:"
  >
    <div
      class="rdp-months"
    >
      <div
        class="rdp-month_wrapper"
      >
        <div
          class="rdp-month_caption"
          id=":r0:-caption-0"
        >
          <span
            aria-live="polite"
            class="rdp-caption_label"
          >
            November 2021
          </span>
        </div>
        <div
          aria-labelledby=":r0:-caption-0"
          aria-multiselectable="false"
          class="rdp-month"
          id=":r0:-grid-0"
          role="grid"
        >
          <div
            aria-rowindex="1"
            class="rdp-weekdays"
            role="row"
          >
            <span
              aria-colindex="1"
              aria-label="Sunday"
              class="rdp-weekday"
              role="columnheader"
            >
              Su
            </span>
            <span
              aria-colindex="2"
              aria-label="Monday"
              class="rdp-weekday"
              role="columnheader"
            >
              Mo
            </span>
            <span
              aria-colindex="3"
              aria-label="Tuesday"
              class="rdp-weekday"
              role="columnheader"
            >
              Tu
            </span>
            <span
              aria-colindex="4"
              aria-label="Wednesday"
              class="rdp-weekday"
              role="columnheader"
            >
              We
            </span>
            <span
              aria-colindex="5"
              aria-label="Thursday"
              class="rdp-weekday"
              role="columnheader"
            >
              Th
            </span>
            <span
              aria-colindex="6"
              aria-label="Friday"
              class="rdp-weekday"
              role="columnheader"
            >
              Fr
            </span>
            <span
              aria-colindex="7"
              aria-label="Saturday"
              class="rdp-weekday"
              role="columnheader"
            >
              Sa
            </span>
          </div>
          <div
            class="rdp-weeks"
            role="rowgroup"
          >
            <div
              aria-rowindex="2"
              class="rdp-week"
              role="row"
            >
              <div
                aria-colindex="1"
                aria-hidden="true"
                class="rdp-day rdp-hidden rdp-outside"
                role="gridcell"
                tabindex="-1"
              >
                31
              </div>
              <div
                aria-colindex="2"
                class="rdp-day"
                role="gridcell"
                tabindex="-1"
              >
                1
              </div>
              <div
                aria-colindex="3"
                class="rdp-day"
                role="gridcell"
                tabindex="-1"
              >
                2
              </div>
              <div
                aria-colindex="4"
                class="rdp-day"
                role="gridcell"
                tabindex="-1"
              >
                3
              </div>
              <div
                aria-colindex="5"
                class="rdp-day"
                role="gridcell"
                tabindex="-1"
              >
                4
              </div>
              <div
                aria-colindex="6"
                class="rdp-day"
                role="gridcell"
                tabindex="-1"
              >
                5
              </div>
              <div
                aria-colindex="7"
                class="rdp-day"
                role="gridcell"
                tabindex="-1"
              >
                6
              </div>
            </div>
            <div
              aria-rowindex="3"
              class="rdp-week"
              role="row"
            >
              <div
                aria-colindex="1"
                class="rdp-day"
                role="gridcell"
                tabindex="-1"
              >
                7
              </div>
              <div
                aria-colindex="2"
                class="rdp-day"
                role="gridcell"
                tabindex="-1"
              >
                8
              </div>
              <div
                aria-colindex="3"
                class="rdp-day"
                role="gridcell"
                tabindex="-1"
              >
                9
              </div>
              <div
                aria-colindex="4"
                class="rdp-day"
                role="gridcell"
                tabindex="-1"
              >
                10
              </div>
              <div
                aria-colindex="5"
                class="rdp-day"
                role="gridcell"
                tabindex="-1"
              >
                11
              </div>
              <div
                aria-colindex="6"
                class="rdp-day"
                role="gridcell"
                tabindex="-1"
              >
                12
              </div>
              <div
                aria-colindex="7"
                class="rdp-day"
                role="gridcell"
                tabindex="-1"
              >
                13
              </div>
            </div>
            <div
              aria-rowindex="4"
              class="rdp-week"
              role="row"
            >
              <div
                aria-colindex="1"
                class="rdp-day"
                role="gridcell"
                tabindex="-1"
              >
                14
              </div>
              <div
                aria-colindex="2"
                class="rdp-day"
                role="gridcell"
                tabindex="-1"
              >
                15
              </div>
              <div
                aria-colindex="3"
                class="rdp-day"
                role="gridcell"
                tabindex="-1"
              >
                16
              </div>
              <div
                aria-colindex="4"
                class="rdp-day"
                role="gridcell"
                tabindex="-1"
              >
                17
              </div>
              <div
                aria-colindex="5"
                class="rdp-day"
                role="gridcell"
                tabindex="-1"
              >
                18
              </div>
              <div
                aria-colindex="6"
                class="rdp-day"
                role="gridcell"
                tabindex="-1"
              >
                19
              </div>
              <div
                aria-colindex="7"
                class="rdp-day"
                role="gridcell"
                tabindex="-1"
              >
                20
              </div>
            </div>
            <div
              aria-rowindex="5"
              class="rdp-week"
              role="row"
            >
              <div
                aria-colindex="1"
                class="rdp-day"
                role="gridcell"
                tabindex="-1"
              >
                21
              </div>
              <div
                aria-colindex="2"
                class="rdp-day"
                role="gridcell"
                tabindex="-1"
              >
                22
              </div>
              <div
                aria-colindex="3"
                class="rdp-day"
                role="gridcell"
                tabindex="-1"
              >
                23
              </div>
              <div
                aria-colindex="4"
                class="rdp-day"
                role="gridcell"
                tabindex="-1"
              >
                24
              </div>
              <div
                aria-colindex="5"
                class="rdp-day rdp-today"
                role="gridcell"
                tabindex="-1"
              >
                25
              </div>
              <div
                aria-colindex="6"
                class="rdp-day"
                role="gridcell"
                tabindex="-1"
              >
                26
              </div>
              <div
                aria-colindex="7"
                class="rdp-day"
                role="gridcell"
                tabindex="-1"
              >
                27
              </div>
            </div>
            <div
              aria-rowindex="6"
              class="rdp-week"
              role="row"
            >
              <div
                aria-colindex="1"
                class="rdp-day"
                role="gridcell"
                tabindex="-1"
              >
                28
              </div>
              <div
                aria-colindex="2"
                class="rdp-day"
                role="gridcell"
                tabindex="-1"
              >
                29
              </div>
              <div
                aria-colindex="3"
                class="rdp-day"
                role="gridcell"
                tabindex="-1"
              >
                30
              </div>
              <div
                aria-colindex="4"
                aria-hidden="true"
                class="rdp-day rdp-hidden rdp-outside"
                role="gridcell"
                tabindex="-1"
              >
                1
              </div>
              <div
                aria-colindex="5"
                aria-hidden="true"
                class="rdp-day rdp-hidden rdp-outside"
                role="gridcell"
                tabindex="-1"
              >
                2
              </div>
              <div
                aria-colindex="6"
                aria-hidden="true"
                class="rdp-day rdp-hidden rdp-outside"
                role="gridcell"
                tabindex="-1"
              >
                3
              </div>
              <div
                aria-colindex="7"
                aria-hidden="true"
                class="rdp-day rdp-hidden rdp-outside"
                role="gridcell"
                tabindex="-1"
              >
                4
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="rdp-nav"
      role="toolbar"
    >
      <button
        aria-controls=":r0:"
        aria-label="Previous Month"
        class="rdp-button_previous"
        name="previous-month"
        type="button"
      >
        <svg
          class="rdp-chevron"
          height="24"
          viewBox="0 0 24 24"
          width="24"
        >
          <polygon
            points="15 17.23 9.43 11.5 15 5.76 13.28 4 6 11.5 13.28 19"
          />
        </svg>
      </button>
      <button
        aria-controls=":r0:"
        aria-label="Next Month"
        class="rdp-button_next"
        name="next-month"
        type="button"
      >
        <svg
          class="rdp-chevron"
          height="24"
          viewBox="0 0 24 24"
          width="24"
        >
          <polygon
            points="8 18.23 13.57 12.5 8 6.76 9.72 5 17 12.5 9.72 20"
          />
        </svg>
      </button>
    </div>
  </div>
</div>
`;
